content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -100,30 +84,54 @@ Each HTTP rule contains the following information:
-  and `servicePort`. Both the host and path must match the content of an incoming request before the
+* A list of paths (for example, `/testpath`), each of which has an associated backend defined with a `service.Name`
+  and a `service.Port.Name` or `service.Port.Number`. Both the host and path must match the content of an incoming request before the
   load balancer directs traffic to the referenced Service.
 * A backend is a combination of Service and port names as described in the
-  [Service doc](/docs/concepts/services-networking/service/). HTTP (and HTTPS) requests to the
+  [Service doc](/docs/concepts/services-networking/service/) or a [custom resource backend](#resource-backends) by way of a CRD. HTTP (and HTTPS) requests to the
   Ingress that matches the host and path of the rule are sent to the listed backend.
 
-A default backend is often configured in an Ingress controller to service any requests that do not
+A defaultBackend is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
-### Default Backend
+### DefaultBackend
 
-An Ingress with no rules sends all traffic to a single default backend. The default
-backend is typically a configuration option of the [Ingress controller](/docs/concepts/services-networking/ingress-controllers) and is not specified in your Ingress resources.
+An Ingress with no rules sends all traffic to a single default backend. The defaultBackend is typically a configuration option
+of the [Ingress controller](/docs/concepts/services-networking/ingress-controllers) and is not specified in your Ingress resources.
 
 If none of the hosts or paths match the HTTP request in the Ingress objects, the traffic is
 routed to your default backend.
 
+### Resource Backends
+
+A _`Resource`_ backend is an ObjectRef to another Kubernetes resource within the same namespace of the Ingress object. A _`Resource`_ is a
+mutually exclusive setting with Service, and will fail validation if both are specified. A common usage for a _`Resource`_ backend is
+to ingress data to a object storage backend with static assets.
+
+{{< codenew file="service/networking/ingress-resource-backend.yaml" >}}
+
+After creating the ingress-resource-backend, you can view it with `kubectl describe ing ingress-resource-backend`
+
+```shell
+Name:             ingress-resource-backend
+Namespace:        default
+Address:
+Default backend:  APIGroup: k8s.example.com, Kind: StorageBucket, Name: static-assets
+Rules:
+  Host        Path  Backends
+  ----        ----  --------
+  *
+              /icons   APIGroup: k8s.example.com, Kind: StorageBucket, Name: icon-assets
+Annotations:  <none>
+Events:       <none>
+```
+
 ### Path Types
 
-Each path in an Ingress has a corresponding path type. There are three supported
+Each path in an Ingress MUST HAVE a corresponding path type. There are three supported
```

Comment:
“MUST HAVE”? It doesn't look like the API lets you not have a path type, so I recommend rewording.

Reply:
Can you recommend how this should be reworded? Should I just say that it is REQUIRED?

Reply:
@cmluciano the reader will want to know if _they_ are REQUIRED to do something special - and if they are, what should that be.

If this were a reference page for people implementing Ingress then it makes sense for the wording to be about specification and requirements. This page is mainly about letting readers know that there is a resource call Ingress and why you might find it useful, so I don't think there is any special action that readers must take.